certs/trade.valorem.xyz.pem

```diff
@@ -0,0 +1,28 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
Is this the Godaddy CA root file?

Reply:
Indeed it is
![Screenshot 2023-06-29 at 23 29 42](https://github.com/valorem-labs-inc/trade-interfaces/assets/35642018/28bededb-6e87-43e5-a550-ddfb4e97ffb4)